kubernetes/product-deployment/scripts/validate-cluster-and-params.sh

```diff
@@ -0,0 +1,60 @@
+isEmpty () {
+    if [ ${#1} -ge 1 ];
+        then 
+            return 0;
+    else
+        return 1;
+    fi;
+}
+
+
+workingdir=$(pwd)
+reldir=`dirname $0`
+cd $reldir
+
+
+isEmpty "${EKS_CLUSTER_NAME}";
+flag=$?
+if [ $flag = 1 ];
+    then echo "EKS_CLUSTER_NAME environment variable is empty."; exit 1
+fi;
+
+isEmpty "${EKS_CLUSTER_REGION}";
+flag=$?
+if [ $flag = 1 ];
+    then echo "EKS_CLUSTER_REGION environment variable is empty."; exit 1
+fi;
+
+isEmpty "${RDS_STACK_NAME}";
+flag=$?
+if [ $flag = 1 ];
+    then echo "RDS_STACK_NAME environment variable is empty."; exit 1
+fi;
+
+isEmpty "${path_to_helm_folder}";
+flag=$?
+if [ $flag = 1 ];
+    then echo "Path to helm folder is empty."; exit 1
+fi;
+
+isEmpty "${product_version}";
+flag=$?
+if [ $flag = 1 ];
+    then echo "Product version is empty."; exit 1
+fi;
+
+isEmpty "${db_engine}";
+flag=$?
+if [ $flag = 1 ];
+    then echo "DB engine value is empty."; exit 1
+fi;
+
+
+# Update kube config file.
+aws eks update-kubeconfig --region ${EKS_CLUSTER_REGION} --name ${EKS_CLUSTER_NAME} || { echo 'Failed to update cluster kube config.';  exit 1; }
```

Comment:
without just saying it's failed. can we print the root cause for the failure?

Reply:
The root cause of the failure will be printed by the update-kubeconfig command output.